<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Rietesh" failures="1" tests="2" name="Test(failed)" time="0.05" errors="0" timestamp="2022-04-05T14:34:41 IST">
  <testcase classname="failedtestcaseExecution.FailedTC" name="login" time="0.008">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at failedtestcaseExecution.FailedTC.login(FailedTC.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="failedtestcaseExecution.FailedTC" name="dashboard" time="0.0">
    <skipped/>
  </testcase> <!-- dashboard -->
  <testcase classname="failedtestcaseExecution.FailedTC" name="logout" time="0.0">
    <ignored/>
  </testcase> <!-- logout -->
  <testcase classname="failedtestcaseExecution.FailedTC" name="profile" time="0.0">
    <ignored/>
  </testcase> <!-- profile -->
</testsuite> <!-- Test(failed) -->
